src/__tests__/index.js

```diff
@@ -0,0 +1,66 @@
+import {createStore} from 'redux';
+import test from 'tape-cup';
+import actionEmitterFunc from '../index.js';
+
+/* Mocks & Mock Factories */
+const getMockEventEmitterFactory = function() {
```

Comment:
can this be part of the UniversalEvents plugin so that we can include that across different plugin tests.

Reply:
I'm just gonna use the native node.js event emitter